otelconf/v0.2.0/config.go

```diff
@@ -143,8 +144,13 @@ func WithOpenTelemetryConfiguration(cfg OpenTelemetryConfiguration) Configuratio
 
 // ParseYAML parses a YAML configuration file into an OpenTelemetryConfiguration.
```

Comment:
I think we should document also the env var substitution somewhere in Go Doc. Here or/and in package documentation, maybe in example test... This should be handled in a separate PR as the package's documentation needs more care anyway: https://pkg.go.dev/go.opentelemetry.io/contrib/otelconf/v0.3.0.

Reply:
I created https://github.com/open-telemetry/opentelemetry-go-contrib/issues/7037

Feel free to resolve

Reply:
Thanks!